### ISSUE TITLE
markdown issue

### ISSUE THREAD

**Issue by triStone** [gitbucket/gitbucket#955]
After updated to gitbucket 3.7 which updated markdown processor, lots of document page are broken, the error message shows as below.
```
java.lang.IndexOutOfBoundsException: Index: 6, Size: 6
	at java.util.ArrayList.rangeCheck(ArrayList.java:635)
	at java.util.ArrayList.get(ArrayList.java:411)
	at io.github.gitbucket.markedj.Parser.tok(Parser.java:81)
	at io.github.gitbucket.markedj.Parser.parse(Parser.java:31)
	at io.github.gitbucket.markedj.Marked.marked(Marked.java:18)
```

**Comment by takezoe** [comment]
Could you show simple markdown example to reproduce this problem?

**Comment by takezoe** [comment]
I expect this issue is same as #949. It was already fixed and will be released with 3.8.
